### PR TITLE
`Programming exercises`: Fix missing translation when creating a programming exercise

### PR DESCRIPTION
**Pull request by tobias-lippert**
#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/guidelines/development-process/#naming-conventions-for-github-pull-requests).
### Motivation and Context
#6356 unified the labels and deleted a lot of unused labels.
When creating a programming exercise, no translation is currently found because the programming-exercise-update component hasn't been changed and still uses the old (now deleted) label.

### Description
Just change the translation key to use the correct (existing label)
### Steps for Testing

Prerequisites:
- 1 Instructor

1. Generate programming exercise
2. Check that the correct heading is shown and **not** translation not found.

#### Exam Mode Testing

Testing is the same as above.

#### Code Review
- [x] Review 1
- [x] Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2
#### Exam Mode Test
- [x] Test

### Test Coverage

unchanged

### Screenshots

image

### CODE REVIEW

**Review by simon-hng — APPROVED** [review]
Code

**Review by YeetTheFirst21 — APPROVED** [review]
Tested on TS1, seems to work as expected both for German and English.

Screenshot 2023-04-03 110557

**Review by Strohgelaender — APPROVED** [review]

**Review by 1999Markus — APPROVED** [review]

**Review by DominikRemo — APPROVED** [review]

File: src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <form name="editForm" role="form" novalidate #editForm="ngForm" (keydown.enter)="isEventInsideTextArea($event)" *ngIf="!isShowingWizardMode">
         <div class="d-flex" *ngIf="!isImport && !programmingExercise.id">
             <div class="flex-grow-1 align-items-center">
-                <h4 *ngIf="!isImport && !programmingExercise.id" id="jhi-programming-exercise-heading-create" jhiTranslate="artemisApp.programmingExercise.home.generateLabel">
+                <h4 *ngIf="!isImport && !programmingExercise.id" id="jhi-programming-exercise-heading-create" jhiTranslate="artemisApp.programmingExercise.home.createLabel">
                     Generate new Programming Exercise
```
Comment (pal03377) [inline src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.html:20]: Shouldn't this then also be `Create Programming Exercise`?

**Review by pal03377 — COMMENTED** [review]
(fixes #6408, I should have looked into the open PRs before 😄 )

**Pull request merged**